### PR TITLE
Add Plans for PostgreSQL by VSHN

### PR DESCRIPTION
**Pull request by glrf**
This PR adds end-user documenation for PostgreSQL plans. The actual feature is still pending in https://github.com/vshn/component-appcat/pull/114. I'd like to have feedback on the UX before I finish that.

The two main things I did that weren't clearly specified and I'd like to have input on:

* For APPUiO Cloud I introduce `standard-X` plans that schedule the instance on `flex` nodes, and `plus-X` plans that schedule the instance on `plus` nodes. Does that make sense? Is that clear?
* I include the sidecar overhead in the plan. So `standard-4` actually only has `3776Mi` memory available for the database. This is confusing, but the alternative is somehow not really better, where we'd bill `4416Mi` for a `standard-4` instance. (And also that way the `standard-4` instance wouldn't even be available with the default resorucequota on APPUiO Cloud)

Finally, I tried to make the documentation a bit less APPUiO Cloud focused, given that we're soon :TM: rolling out AppCat on APPUiO managed. I'm not sure if that made things more confusing

### CODE REVIEW

File: docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+----
+
+Please consult the Kubernetes documentation on https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/[how to assign workloads to nodes using node selectors].
+
+
+[WARNING,caption="APPUiO Cloud"]
+====
+On APPUiO Cloud, this can be used to schedule instances on specific https://docs.appuio.cloud/user/references/node-classes.html[Node Classes].
+
+However, this can also be done by choosing the `standard-x` or `plus-X` plans, which will schedule the instance on `flex` or `plus` nodes.
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc:154]: Nit: I'd probably mention this in the `available plans` section of the docs.

**Review by Kidswiss — APPROVED** [review]
I like that this cleans up the example quite a bit. People were adding every single new feature to the example which made it quire bloated IMHO.

With this it's much more simple again.

I like the approach with the overhead.

File: docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+* There is a `plus-X` variant of each plan that will schedule the instance on a https://docs.appuio.cloud/user/references/node-classes.html[Plus Node].
+* The `standard-8` plan is not available as it requests more memory than the https://docs.appuio.cloud/user/references/default-quota.html[Default Quota] allows.
+ If you need a larger instance, please contact https://docs.appuio.cloud/user/contact.html[APPUiO Cloud support] and manually request more memory according to <<_custom_sizing>>.
+====
+
+
+== Custom Sizing
+
+While we provide some pre-configured plans, you are free to choose custom values for CPU, memory, and disk size.
+To do that simply set `spec.parameters.size.cpu`, `spec.parameters.size.memory`, or `spec.parameters.size.disk`.
```
Comment (lieneluksika) [inline docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc:71]: Can a customer choose also between flex or plus nodes here?

**Review by lieneluksika — APPROVED** [review]

File: docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc (hunk 2 above)
Comment (glrf) [reply docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc:71]: No. This done through the `scheduling` field. But I think I'll need to write this a bit more clearly

**Review by akosma — APPROVED** [review]

**Pull request merged**